### ISSUE TITLE
Text in code ticks does not show up as a link

### ISSUE THREAD

**Issue by Nadine2016** [colinwilson/lotusdocs#34]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

When embedding a link in code ticks, the link will show up as code, but not as a link. 

Sample code: 
```
[`mylink`](www.example.com)
```

Output: 

<img width="160" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/17709352/e2edab05-26e4-46c9-ace5-7d43e8bbd3ec">

If we add more text to the link, the text will display as a link, but the text in code ticks will not

<img width="222" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/17709352/f9272789-015c-4da2-91a1-da3dfb46f4e5">

### Expected behavior

Text in code ticks should display with mono font, but still be colored as a link. 

### Environment

```markdown
- OS: MacOS
- Browser: Chrome
- Browser Version: 116
```

**Comment by colinwilson** [comment]
> Text in code ticks should display with mono font, but still be colored as a link.

Agree 👍

**Comment by colinwilson** [comment]
Fixed by 623dadb

**Comment by Nadine2016** [comment]
thank you so much!
